drivers/opt3001/include/opt3001_regs.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_opt3001
+ *
+ * @{
+ * @file
+ * @brief       Register definitions for OPT3001 devices
+ *
+ * @author      Jannes Volkens <jannes.volkens@haw-hamburg.de>
+ */
+
+#ifndef OPT3001_REGS_H
+#define OPT3001_REGS_H
+
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name  OPT3001 registers
+ * @{
+ */
+ #define OPT3001_REGS_RESULT             0x00 /**< Result register */
+ #define OPT3001_REGS_CONFIG             0x01 /**< Configuration register */
+ #define OPT3001_REGS_LOW_LIMIT          0x02 /**< Low Limit register */
+ #define OPT3001_REGS_HIGH_LIMIT         0x03 /**< High Limit register */
+ #define OPT3001_REGS_MANUFACTURER_ID    0x7E /**< Manufacturer ID register */
+ #define OPT3001_REGS_DEVICE_ID          0x7F /**< Device ID register */
+
+ #define OPT3001_DID_VALUE    (0x3001) /**< Device ID value */
+
+ #define OPT3001_CONFIG_RESET    (0xC810) /**< Reset value */
+
+ #define OPT3001_CONFIG_RN_FSR    (0xC000) /**< Automatic full-scale setting mode */
```

Comment:
While you fix the typo, you can also align the indentation of those three properly like the defines above :wink: